packages/tpetra/core/example/BlockCrs/Tpetra_TestBlockCrsMeshDatabase.hpp

```diff
@@ -594,10 +594,10 @@ namespace BlockCrsTest {
       const auto num_owned_elements = _owned_gids.extent(0);
       const auto num_remote_elements = _remote_gids.extent(0);
 
-      const auto owned_first= &_owned_gids(0);
-      const auto owned_last = &_owned_gids(num_owned_elements-1);
-      const auto remote_first = &_remote_gids(0);
-      const auto remote_last = &_remote_gids(num_remote_elements-1);
+      const auto owned_first= _owned_gids.data();
```

Comment:
What is going on here?  Is this a null pointer or something?

Reply:
This is generally good practice.  Both std::vector and Kokkos::View (and std::span etc.) have a `.data()` method.  I suppose it could be possible for some process to own zero GIDs, though examples like this should (if they don't already) exclude processes that don't fit evenly into the division of elements.

Reply:
@bartlettroscoe its not necessarily a null pointer, the problem is that just executing `_owned_gids(0)` assumes that there is at least one entry, and triggers an out-of-bounds error otherwise.